Assets/PatchKit/UnityEditorExtension/Editor/UI/ConnectAppScreen.cs

```diff
@@ -0,0 +1,299 @@
+using System.Linq;
+using JetBrains.Annotations;
+using PatchKit.Api.Models.Main;
+using PatchKit.UnityEditorExtension.Core;
+using UnityEditor;
+using UnityEngine;
+using System.Collections;
+using UnityEngine.Assertions;
+using System.Collections.Generic;
+
+namespace PatchKit.UnityEditorExtension.UI
+{
+    public class ConnectAppScreen : Screen
+    {
+        public class LinkedResult
+        {
+            public readonly App App;
+
+            public LinkedResult(App app)
+            {
+                App = app;
+            }
+        }
+
+        #region GUI
+
+        public override string Title
+        {
+            get { return "Connect App"; }
+        }
+
+        public override Vector2? Size
+        {
+            get { return new Vector2(400f, 600f); }
+        }
+
+        public override void UpdateIfActive()
+        {
+            if (!Config.GetLinkedAccountApiKey().HasValue)
+            {
+                Push<NotLinkedAccountScreen>().Initialize();
+            }
+        }
+
+        public override void Draw()
+        {
+            EditorGUILayout.BeginHorizontal();
+            {
+                if (GUILayout.Button(
+                    new GUIContent((Texture) EditorGUIUtility.Load("arrowIcon.png"),
+                                   "Return"),
+                    GUILayout.Width(35),
+                    GUILayout.Height(20)))
+                {
+                    Dispatch(() => Cancel());
+                }
+
+                GUILayout.Space(220);
+                if (GUILayout.Button(
+                    "Create new app",
+                    GUILayout.Width(130),
+                    GUILayout.Height(20)))
+                {
+                    Dispatch(() => CreateNew());
+                }
+            }
+            EditorGUILayout.EndHorizontal();
+            EditorGUILayout.BeginHorizontal();
+            {
+                GUILayout.FlexibleSpace();
+                GUILayout.Label(
+                    string.Format(
+                        "Connect your PatchKit application for {0}.",
+                        _platform.ToDisplayString()),
+                    EditorStyles.boldLabel);
+                GUILayout.FlexibleSpace();
+            }
+            EditorGUILayout.EndHorizontal();
+
+            ShouldFilterByPlatform = EditorGUILayout.Toggle(
+                "Filter apps by platform",
+                ShouldFilterByPlatform);
+            EditorGUILayout.Space();
+
+            _scrollViewVector = EditorGUILayout.BeginScrollView(
+                _scrollViewVector,
+                EditorStyles.helpBox);
+
+
+            int appIndex = 0;
+            foreach (App app in Apps)
+            {
+                DrawApp(app, appIndex % 2);
+                appIndex++;
+            }
+
+            EditorGUILayout.EndScrollView();
+        }
+
+        private void DrawApp(App app, int i)
```

Comment:
What is this `int i`?

Reply:
I see it used below on lines 107-109

```
Color backgroundColor = (i == 0)
    ? new Color(1f, 1f, 1f)
    : new Color(0.9f, 0.9f, 0.9f);
```

and nowhere else I think

Reply:
I agree with Jakub, more descriptive name would be good.